cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DistributedStorageProviderNamespaceAdmin.java

```diff
@@ -62,6 +62,9 @@
   public void create(NamespaceMeta namespaceMeta) throws IOException, ExploreException, SQLException {
     // create filesystem directory
     super.create(namespaceMeta);
+    if (NamespaceId.DEFAULT.equals(namespaceMeta.getNamespaceId())) {
```

Comment:
This is because we do not want to attempt to create/delete the default namespace in HBase/Hive. It is an unnecessary operation that will always fail.